pkg/client/logs/logs.go

```diff
@@ -67,80 +66,38 @@ func (l *logClient) List(ctx context.Context, args ...interface{}) (interface{},
 		}
 	}
 
-	// Get the last log from the list of logs for the previous day.
-	// This is used as the starting point for the fetching of the logs.
-	// This allows us to use the checkpoint pagination style
-	var checkpoint *management.Log
-	var err error
-	checkpoint, err = l.findLatestCheckpoint(ctx, from, 0, 30)
-	switch {
-	case errors.Is(err, errLastCheckpointMaxAttemptsReached):
-		// do nothing
-	case errors.Is(err, errors.QuotaLimitExceeded):
-		return allLogs, ErrAPIRateLimitReached
-	case err != nil:
-		return allLogs, err
-	}
-
 	for {
-		logs, err := l.fetchLogs(ctx, checkpoint)
+		query := fmt.Sprintf("date:{%s TO *]", from.UTC().Format(time.RFC3339))
+
+		logs, err := l.mgmt.List(
+			ctx,
+			management.IncludeFields("type", "log_id", "date", "client_name"),
+			management.Query(query),
+			management.Sort("date:1"),
+			management.Take(50), // max number of items allowed by Auth0
```

Comment:
> Ok, let's be honest: I tried to search online but I didn't find that `50` limit anywhere. But still, for some reason, that's the maximum number of items returned for each API call by Auth0
> 
> @excavador your suggestion to make them be constants is very good, so let me do that and add a small comment on top of it to explain that it's a sensible value
> 
> Edit: done in [8d45e33](https://github.com/tfadeyi/auth0-simple-exporter/commit/8d45e33155da119ee70042a4fcf504395bc71d80)

You can use the Management API to retrieve up to 100 log events per request using the [/get_logs](https://auth0.com/docs/api/management/v2/#!/Logs/get_logs) endpoint, which supports two types of consumption:

Reply:
@excavador that's exactly the API I'm using, but unfortunately I'm getting only `50` log lines :sweat_smile: look

I tried setting `take=100` as per the link you posted:

![image](https://github.com/tfadeyi/auth0-simple-exporter/assets/49731484/b49f35d3-4496-4a0d-91a1-166f3da26254)

And this is the result:

![image](https://github.com/tfadeyi/auth0-simple-exporter/assets/49731484/f4b2e4e4-3f4b-4eee-9352-e07b9f7eaa4d)

But guys, is this actually a problem? 50 log lines are a lot, and I think it's better to stick to this number even if only some users are experiencing this. Otherwise the `logs.List` algorithm wouldn't work well

Reply:
@tfadeyi @excavador I found it! Here: https://auth0.com/docs/troubleshoot/product-lifecycle/past-migrations/migrate-to-paginated-queries

> After 26 January 2021 (or Private Cloud version 2202), requests to Management API v2 endpoints will return a maximum of 50 items for tenants in the Public Cloud

I'm gonna put a comment with this link in the code